modules/tracking/samples/multiTracker_dataset.cpp

```diff
@@ -227,4 +234,11 @@ int main(int argc, char *argv[])
 	waitKey(0);
 
 	return 0;
-}
\ No newline at end of file
+}
+
+#else // ! HAVE_OPENCV_DATASETS
+int main() {
+	// !!! this sample requires opencv_datasets !!!
```

Comment:
It's better to place CV_Error() call here because not everyone looks at sources before running samples.